integrations/install_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package integration
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"os/user"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"code.gitea.io/gitea/integrations/internal/utils"
+)
+
+// The HTTP port listened by the Gitea server.
+const ServerHTTPPort = "3001"
+
+const _RetryLimit = 10
+
+func makeSimpleSettings(user, workdir, port string) map[string][]string {
+	return map[string][]string{
+		"db_type":        {"SQLite3"},
+		"db_host":        {"localhost"},
+		"db_path":        {workdir + "data/gitea.db"},
```

Comment:
The 'workdir' is created by ioutil.TempDir if not specified (that is, an empty string). 

My comment regarding the field WorkDir is outdated and does not match with the reality. I'll fix it. Thanks.